packages/queries/src/getColumns.ts

```diff
@@ -0,0 +1,33 @@
+import { DbSchema } from './types/schemas';
+
+export function getColumns(schema: DbSchema, table: string) {
+    const select: Record<string, true> = {};
+
+    const properties = schema.properties;
+
+    const tableDef = properties[table];
+
+    const tableDefProperties = tableDef.properties;
+
+    const columnsDef = tableDefProperties.columns;
+
+    const columnsDefProperties = columnsDef.properties;
+
+    for (const column in columnsDefProperties) {
+        if (column) {
+            const columnDef = columnsDefProperties[column];
+
+            const columnDefProperties = columnDef.properties;
+
+            const selectableDef = columnDefProperties.selectable;
```

Comment:
There's no need to declare `selectableDef`. This is just adding an extra line of code with no purpose. 

const col

As a general rule of thumb don't declare additional variables unless there's a reason for it.